{% extends "layout.html" %}

<!-- Header -->
{% block header %}
<header class="py-5 mb-5" id="header" style="background-image: linear-gradient(0deg, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%),url(https://static.igem.wiki/teams/4214/wiki/engineering/header-eng-success.png);">
  <div class="container h-100">
    <div class="row h-100">
      <div class="col-lg-12">
        <h1 class="display-4 text-white">Engineering Success</h1>
      </div>
    </div>
  </div>
</header>
{% endblock %}

{% block page_content %}

<!-- side bar -->
<div class="row mt-4">
  <div class="col-lg-3">
    <div class="side-bar sticky-top">
      <nav class="navigation">
        <ul class="nav-ul-sidebar">
          <li class="nav-ul-li-sidebar">
            <a class="nav-ul-li-a-sidebar" href="#tag1">Introduction</a>
            <a class="nav-ul-li-a-sidebar" href="#tag2">Engineering Cycle - Phase I</a>
            <a class="nav-ul-li-a-sidebar" href="#tag3">Engineering Cycle - Phase II</a>
            <a class="nav-ul-li-a-sidebar" href="#tag4">Engineering Cycle - Phase III</a>
            <a class="nav-ul-li-a-sidebar" href="#tag5">Engineering Cycle - Phase IV</a>


          </li>
        </ul>
      </nav>
    </div>
  </div>
<!-- END of sidebar -->

  <!-- Include chapters here -->
  <div class="col-lg-9">
    <div class="main-content">

      <section id="tag1">
        <h2>Introduction</h2>
            <p>Our goal was to engineer a recombinant intrinsic factor (called “BiG-IF”) that the autoantibodies against pernicious anaemia (PA) cannot bind to, while maintaining its vitamin B12 affinity through site-directed mutagenesis (SDM). Unlike other systems, a biological system is more complex and challenging to engineer and with such systems, often our understanding is limited and the pace is reduced. Thus, it is important to have a comprehensive process with a very tight-knit feedback loop. This was accomplished by executing the carefully curated DBTL engineering cycle, consisting of design, build, test and learn stages, while running several feedback loops in parallel.</p>

            <figure class="reveal">
              <img src="https://static.igem.wiki/teams/4214/wiki/engineering/engineering-cycle.png" class="img-fluid reveal" alt="engineering cycle">
              <figcaption>
                <p><i style="font-weight:bold;">Figure.</i><i> Engineering cycle representing the design, build, test and learn stages</i></p>
              </figcaption>
            </figure>
      </section>

      <section id="tag2">
        <h2 class="revealfromtop">Engineering Cycle - Phase I</h2>

        <h3 class="revealfromtop">Design</h3>

          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-lightbulb"></i> After a thorough literature review, IF mutants were designed using the BLOSUM62 substitution matrix to preserve the wild type structure as much as possible.<br><br>
            <i class="fa fa-solid fa-lightbulb"></i> IF being a mammalian protein, a display vector, pDisplayTM (display vector) from Thermo Fisher was selected as the transfection vector.<br><br>               
            <i class="fa fa-solid fa-lightbulb"></i> A production vector, 1SB2C3, was also chosen to enable the production of the protein in a bacterial system and for downstream purification and isolation.<br><br>          
            <i class="fa fa-solid fa-lightbulb"></i> Wet lab experiments were designed to clone and amplify the wild type plasmids.<br><br> 
          </p>

        <h3 class="revealfromtop">Build</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-flask"></i> The mutant structures were built using AlphaFold.<br><br>
            <i class="fa fa-solid fa-flask"></i> The IF gene was cloned into the display and production vectors using restriction cloning.<br><br>
            <i class="fa fa-solid fa-flask"></i> The SDM was initiated for the mutants in phases.<br><br>
          </p>

        <h3 class="revealfromtop">Test</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-dna"></i> Of all the structures built, the improperly folded mutants were excluded and the rest were further subjected to in-silico modelling.<br><br>
            <i class="fa fa-solid fa-dna"></i> To assess the strength of binding, docking studies were carried out using Autodock Vina and Mutabind2 for B12 and cubilin receptor respectively.<br><br>
            <i class="fa fa-solid fa-dna"></i> Immunogenicity studies were performed to check for introduction of novel epitopes in the mutants.<br><br>
            <i class="fa fa-solid fa-dna"></i> The sequencing results from cloning of the display vector, revealed that IF had indeed been inserted into the expression vector but had an unintended stop codon after the IF sequence but right before the PDGFR sequence.<br><br>
          </p>

        <h3 class="revealfromtop">Learn</h3>      
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-book"></i> Mutants binding affinities to both B12 and cubilin receptor, which are essential for the internalization of the B12-IF complex, were evaluated along with the immunogenicity scores and the most promising candidates were selected to be tested in the wet lab.<br><br>
            <i class="fa fa-solid fa-book"></i> The stop codon in the sequencing results of all the clones and the wild type resulted from the improper amplification of the IF gene fragment prior to the insertion of the newer restriction sites for cloning. This had to be removed to ensure a successful display of the protein.<br><br>
            <i class="fa fa-solid fa-book"></i> The mutations were unsuccessful in the first attempt at SDM as the primer design did not align with the SDM kit we were using.<br><br>
            <i class="fa fa-solid fa-book"></i> In our attempt to produce the mammalian protein, we chose a bacterial system that is proficient at production but it, theoretically, lacked an ability to produce a fully folded protein that can be successfully isolated and purified. This prompted us to pick a new plasmid, and a new strategy to bring the protein out into the periplasm using a PelB leader sequence.<br><br>
          </p>
      </section>
      
      <section id="tag3" style="margin-bottom:30px;">
        <h2 class="revealfromtop">Engineering Cycle - Phase II</h2>

        <h3 class="revealfromtop">Design</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-lightbulb"></i> A new set of primers were designed carefully to possess the necessary overlapping bases at and around the mutation site.<br><br> 
            <i class="fa fa-solid fa-lightbulb"></i> A new primer pair was designed to remove the stop codon from the expression vector.<br><br>               
            <i class="fa fa-solid fa-lightbulb"></i> Wet lab experiments were designed to clone, mutate and amplify the wild type and the mutant plasmids.<br><br>          
            <i class="fa fa-solid fa-lightbulb"></i> Transfection experiments were designed for HEK293 suspension culture.<br><br> 
          </p>

        <h3 class="revealfromtop">Build</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-flask"></i> The mutations were incorporated into the expression vector in iterative steps from single to triple mutations with varying levels of progress.<br><br>
            <i class="fa fa-solid fa-flask"></i> The stop codon from all the mutants were removed from the plasmids using SDM and the plasmids were plasmid prepped to ensure adequate concentration needed for transfection.<br><br>
            <i class="fa fa-solid fa-flask"></i> Recombinant production of IF from BL21 cells was attempted with 2 different concentrations of IPTG with limited success.<br><br>
            <i class="fa fa-solid fa-flask"></i> The mutant vectors were transfected into HEK293F, suspension cells.<br><br>
          </p>

        <h3 class="revealfromtop">Test</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-dna"></i> The sequencing of the mutations revealed that the SDMs were successful with the desired substitutions.<br><br>
            <i class="fa fa-solid fa-dna"></i> The transfection efficiency appeared to be too low with minimal GFP being expressed.<br><br>
            <i class="fa fa-solid fa-dna"></i> The results on the Western blot (WB) revealed that the IF is expressed on the surface as well as in the cytoplasm, which was undesirable as we needed maximum IF to be displayed on the surface.<br><br>
            <i class="fa fa-solid fa-dna"></i> The flow cytometry results proved to be inconclusive.<br><br>
            <i class="fa fa-solid fa-dna"></i> The recombinant production of IF in bacteria did not yield any definite band in the size range expected, hence warranting a change in strategy.<br><br>
          </p>

        <h3 class="revealfromtop">Learn</h3>      
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-book"></i> The WB results confirmed that there was a band in the size range that we were expecting, but without any controls nor double antibody binding, we could not reliably conclude the results.<br><br>
          </p>
      </section>


      <section id="tag4" style="margin-bottom:30px;">
        <h2 class="revealfromtop">Engineering Cycle - Phase III</h2>
        <h3 class="revealfromtop">Design</h3>

          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-lightbulb"></i> A Pet26b+ plasmid was chosen to clone the IF gene containing the His-tag into and lead it into the periplasm.<br><br> 
            <i class="fa fa-solid fa-lightbulb"></i> A positive control of the pure IF protein for the SDS-Page and WB was ordered.<br><br>               
            <i class="fa fa-solid fa-lightbulb"></i> The WB was planned to run with antibodies against the IF protein and the Myc tag which immediately superseded the IF insertion in the expression vector.<br><br>          
          </p>

        <h3 class="revealfromtop">Build</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-flask"></i> The display vector was transfected into HEK293 adherent cells and fluorescent WB and fluorescence microscopy (FM) was performed.<br><br>
            <i class="fa fa-solid fa-flask"></i> The protein was produced using the new production vector against the previous vector in three different inducer concentrations.<br><br>
          </p>

        <h3 class="revealfromtop">Test</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-dna"></i> The WB revealed that there was indeed a production of the variant 20201 but failed to show any wild type production.<br><br>
            <i class="fa fa-solid fa-dna"></i> The HEK293 adherent cells failed to adhere firmly to cover slips thereby not yielding any FM imaging.<br><br>
            <i class="fa fa-solid fa-dna"></i> There was a band at the size range in the bacterial production, but it was getting eluted along with the other proteins at too low a concentration of imidazole.<br><br>
          </p>

        <h3 class="revealfromtop">Learn</h3>      
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-book"></i> Poly-D-Lysine was utilized to coat the wells and the coverslips and FM was reattempted.<br><br>
            <i class="fa fa-solid fa-book"></i> A new wild type protein was used to generate a band in the desired size range.<br><br>
            <i class="fa fa-solid fa-book"></i> The wash buffer was diluted to ensure it left the IF intact in the column.<br><br>
          </p>
      </section>


      <section id="tag5" style="margin-bottom:30px;">
        <h2 class="revealfromtop">Engineering Cycle - Phase IV</h2>

        <h3 class="revealfromtop">Design</h3>

          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-lightbulb"></i> The protein production was initiated with 0.5mM of inducer as only that concentration yielded a band.<br><br> 
            <i class="fa fa-solid fa-lightbulb"></i> Transfection experiments were initiated again with the poly-D-lysine coating of wells.<br><br>               
          </p>

        <h3 class="revealfromtop">Build</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-flask"></i> The display vector was once again transfected into HEK293 adherent cells that were firmly attached onto the coating and FM was performed.<br><br>
            <i class="fa fa-solid fa-flask"></i> The production of IF was re-initiated with a wash buffer concentration of 0.5 mM and an elution buffer concentration of 15mM imidazole.<br><br>
          </p>

        <h3 class="revealfromtop">Test</h3>
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-dna"></i> The FM results revealed that there was expression of IF on the surface of cells.<br><br>
            <i class="fa fa-solid fa-dna"></i> The images were preserved and taken forward for image analyses.<br><br>
            <i class="fa fa-solid fa-dna"></i> This time, the protein failed to bind to the resin.<br><br>

          </p>

        <h3 class="revealfromtop">Learn</h3>      
          <p class="reveal" style="line-height: 1.2;">
            <i class="fa fa-solid fa-book"></i> Needs a re-evaluation and optimization of the protocol.<br><br>
          </p>
      </section>


      <section>
        <div class="box">
          <a href="{{ url_for('pages', page='description') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages"><i class="fa fa-solid fa-arrow-left fa_pages"></i> Description</p></div></a>

          <a href="{{ url_for('pages', page='dry-lab') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages">Dry lab <i class="fa fa-solid fa-arrow-right fa_pages"></i></p></div></a>
        </div>
      </section>  
    </div>
  </div>
</div>
{% endblock %}
