{% extends "layout.html" %}

<!-- Header -->
{% block header %}
<header class="py-5 mb-5" id="header" style="background-image: linear-gradient(0deg, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%),url(https://static.igem.wiki/teams/4214/wiki/model/model-header.png);">
  <div class="container h-100">
    <div class="row h-100">
      <div class="col-lg-12">
        <h1 class="display-4 text-white">Model</h1>
      </div>
    </div>
  </div>
</header>
{% endblock %} 

{% block page_content %}

<!-- side bar -->
<div class="row mt-4">
  <div class="col-lg-3">
    <div class="side-bar sticky-top">
      <nav class="navigation">
        <ul class="nav-ul-sidebar">
          <li class="nav-ul-li-sidebar">
            <a class="nav-ul-li-a-sidebar" href="#tag1">Mutant creation</a>

            <a class="nav-ul-li-a-sidebar" href="#tag2">Docking and MD</a>
              <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag2_1">Docking</a>
              <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag2_2">Molecular  Dynamics</a>
              <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag2_3">Immunogenicity </a>

            <a class="nav-ul-li-a-sidebar" href="#references">References</a>
          </li>
        </ul>
      </nav>
    </div>
  </div>
<!-- END of sidebar -->

  <!-- Include chapters here -->
  <div class="col-lg-9">
    <div class="main-content">

      <section id="tag1">
        <h2 class="revealfromtop">Mutant creation</h2>
        <p class="reveal">To address the challenging task of protein design, we performed thorough modelling to select the best candidates to be tested in the lab.</p>

        <p class="reveal">The first step in the process of engineering an Intrinsic Factor (IF) that can evade the autoantibodies is to identify the epitope on the IF and engineer an IF that can evade the clearance mediated by autoantibodies. Based on previous knowledge <a href="#references">[1]</a>, the residues in the alpha helix region adjacent to the cobalamin (vitamin B12) binding pocket:  256Ser, 258Lys, 262Tyr and 265Val were to be mutated. Optimal mutations to substitute the residues with other or “similar” residues were decided and engineered in silico.</p>

        <figure>
          <img src="https://static.igem.wiki/teams/4214/wiki/model/model.gif" class="img-fluid reveal" alt="Structure of IF + B12 + cubulin with amino acids highlighted">
          <figcaption>
            <p><i style="font-weight:bold;">Model.</i><i> Structure of IF + B12 + cubulin with amino acids highlighted</i></p>
          </figcaption>
        </figure>
        
        <p class="reveal">Then, we designed our mutants based on the rationale of preserving structure as much as possible. This principle was chosen as the antibody binding region is in close proximity to the B12 binding region. Thus, mutagenesis should prevent IF from binding to autoimmune antibodies while preserving ligand-receptor binding affinity. For this purpose, we used BLOSUM62 matrix <a href="#references">[2]</a> to detect closely related amino acids. These are the ones whose substitution is well tolerated in evolution.</p>

        <p class="reveal">We performed protein structure predictions to verify that the designed mutants would preserve the properties of the wild-type intrinsic factor. We utilised a well-known tool called AlphaFold2 <a href="#references">[3]</a> and used it as ColabFold <a href="#references">[4]</a>. Structures of all 21 mutants were obtained as PDB files, visualised and structurally aligned to the wild-type IF crystallography structure (PDB ID: 2pmv) using PyMol  <a href="#references">[5]</a>. This enabled us to refine the list of tested mutants excluding the improperly folded ones.</p>        
        
        <!--carousel for 17 pictures -->
          <div class="container center reveal">
            <div id="carouselExampleIndicators4" class="carousel carousel-light slide" data-interval="false">
              <div class="carousel-indicators">
                <button type="button" data-bs-target="#carouselExampleIndicators3" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="1" aria-label="Slide 2"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="2" aria-label="Slide 3"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="3" aria-label="Slide 4"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="4" aria-label="Slide 5"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="5" aria-label="Slide 6"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="6" aria-label="Slide 7"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="7" aria-label="Slide 8"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="8" aria-label="Slide 9"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="9" aria-label="Slide 10"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="10" aria-label="Slide 11"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="11" aria-label="Slide 12"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="12" aria-label="Slide 13"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="13" aria-label="Slide 14"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="14" aria-label="Slide 15"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="15" aria-label="Slide 16"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="16" aria-label="Slide 17"></button>
                <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="17" aria-label="Slide 18"></button>
              </div>

              <div class="carousel-inner">
                <div class="carousel-item active">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256a-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256t-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/k258r-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/y262h-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/y262w-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/y262f-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/v265i-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256tk258r-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256ty262f-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256tv265i-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/k258ry262f-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/k258rv265i-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/y262fv265i-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256tk258ry262f-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256tk258r265i-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256ty262v265i-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/l258ry262fv265i-mut-view-1.png" class="img-fluid"/>
                </div>
                <div class="carousel-item">
                  <img loading="lazy" src="https://static.igem.wiki/teams/4214/wiki/model/carousel/s256tk258ry262fv265i-mut-view-1.png" class="img-fluid"/>
                </div>                
              </div>

              <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleIndicators4" data-bs-slide="prev">
                <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                <span class="visually-hidden">Previous</span>
              </button>
              <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleIndicators4" data-bs-slide="next">
                <span class="carousel-control-next-icon" aria-hidden="true"></span>
                <span class="visually-hidden">Next</span>
              </button>
            </div>
          </div>
          
      </section>



      <section id="tag2">
        <h2 class="revealfromtop">Docking and MD</h2>
      </section>

      <section id="tag2_1">
        <h3 class="revealfromtop">Docking</h3>
        <p class="reveal">The final step of the modelling consisted of screening for potent therapeutic candidates. We performed docking studies to ensure that the binding affinity of engineered protein is at least on par with the wild-type IF.  Docking studies were performed in 2 steps. First, we predicted the binding energy. AutoDock Vina <a href="#references">[6]</a> tool was selected for this task as it is a well-established tool for docking simulations. Energy refinement was performed using NNscore 2.0 neural network <a href="#references">[7]</a>. The NNScore 2.0 scoring function was used to estimate the binding energy and dissociation constant, as it is more accurate than the AutoDock Vina scoring function.</p>

        <table>
          <tr>
              <td><h6>Mutant</h6></td>
              <td><h6>energy, kcal/mol<h6></td>
              <td><h6>Kd, pM<h6></td>
          </tr>
          <tr>
              <td>WT</td>
              <td>-14.17</td>
              <td>929.39</td>
          </tr>
          <tr>
              <td>S256A</td>
              <td>-14.64</td>
              <td>270.53</td>
          </tr>
          <tr>
              <td>S256T</td>
              <td>-14.22</td>
              <td>342.33</td>
          </tr>
          <tr>
              <td>K258R</td>
              <td>-16.41</td>
              <td>111.74</td>
          </tr>
          <tr>
              <td>Y262H</td>
              <td>-14.13</td>
              <td>66.03</td>
          </tr>
          <tr>
              <td>Y262W</td>
              <td>-14.2</td>
              <td>116.91</td>
          </tr>
          <tr>
              <td>Y262F</td>
              <td>-15.08</td>
              <td>112.11</td>
          </tr>
          <tr>
              <td>V265I</td>
              <td>-15</td>
              <td>215.17</td>
          </tr>
          <tr>
              <td>S256TK258R</td>
              <td>-16.04</td>
              <td>53.32</td>
          </tr>
          <tr>
              <td>S256TY262F</td>
              <td>-14.75</td>
              <td>245.04</td>
          </tr>
          <tr>
              <td>S256TV265I</td>
              <td>-16.49</td>
              <td>72.19</td>
          </tr>
          <tr>
              <td>K258RY262F</td>
              <td>-13.99</td>
              <td>41.78</td>
          </tr>
          <tr>
              <td>K258RV265I</td>
              <td>-13.92</td>
              <td>313.14</td>
          </tr>
          <tr>
              <td>Y262FV265I</td>
              <td>-15.26</td>
              <td>144.1</td>
          </tr>
          <tr>
              <td>S256TK258RY262F</td>
              <td>-13.65</td>
              <td>127.77</td>
          </tr>
          <tr>
              <td>S256TK258RV265I</td>
              <td>-14.22</td>
              <td>100.04</td>
          </tr>
          <tr>
              <td>S256TY262FV265I</td>
              <td>-14.33</td>
              <td>63.77</td>
          </tr>
          <tr>
              <td>K258RY262FV265I</td>
              <td>-16.2</td>
              <td>102.57</td>
          </tr>
          <tr>
              <td>S256TK258RY262FV265I</td>
              <td>-14.44</td>
              <td>50.69</td>
          </tr>
        </table>

        <h3></h3>

        <!-- dropdowns of model -->
        <div class="row">
          <div class="col-md-3 form-floating reveal">
            <form id="form1" method="post">
              <label class="label-dropdown" for="floatingSelect">Model 1:</label>
              <select class="form-select" id="model1" onchange="setIframeSource_model1()">
                <option value="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/kth-logo.jpg">KTH</option> 
                <option value="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/ki.png">KI</option> 
                <option value="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/igem-logo.png">iGEM</option> 
              </select>
            </form>
          </div>

          <div class="col-md-3"></div>

          <div class="col-md-3 form-floating reveal">
            <form id="form1" method="post">
              <label class="label-dropdown" for="floatingSelect">Model 2:</label>
              <select class="form-select" id="model2" onchange="setIframeSource_model2()">
                <option value="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/ki.png">KI</option> 
                <option value="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/kth-logo.jpg">KTH</option> 
                <option value="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/igem-logo.png">iGEM</option> 
              </select>
            </form>
          </div>
          
          <div class="col-md-3"></div>

        </div>
        
        <div class="row"> <!-- Just for some space between toggles and images! -->
          <div class="col-md-3"></div>
          <div class="col-md-3"></div>
          <div class="col-md-3"></div>
          <div class="col-md-3"></div>
        </div>
        
        <div class="row">
          
          <div class="col-md-3">
            <p align="center" class="reveal"><iframe id="myModel1" title="Model 1" width="300" height="300" src="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/kth-logo.jpg" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>
            </p>
          </div>

          <div class="col-md-3"></div>

          <div class="col-md-3">
            <p align="center" class="reveal"><iframe id="myModel2" title="Model 2" width="300" height="300" src="https://static.igem.wiki/teams/4214/wiki/experiments/attributions/ki.png" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>
            </p>
          </div>

          <div class="col-md-3"></div>

        </div>
        

      <p class="reveal">The most promising candidates were selected to be tested in wet lab experiments. We found that most proposed mutants have similar or better binding affinity to vitamin B12. Therefore, those results indicate that our mutant selection pipeline is adequate and applicable to the research question.</p>
      </section>


      <section id="tag2_2">
        <h3 class="revealfromtop">Molecular  Dynamics</h3>
   
        <p class="reveal">To have a closer look at the interactions at molecular level, attempts were made to model the IF-B12, protein-ligand complex for molecular dynamics study using GROMACS <a href="#references">[8]</a> software. Some of our initial tasks included pre-processing of the available crystallography structure data and topology generation. As the available structure is not fully resolved for the entire protein, a tool called Swiss-PDB viewer <a href="#references">[9]</a> was used to fill in the missing atoms (predict structure) and the structure was cleaned of glycosylation and other unwanted atoms were removed using Chimera <a href="#references">[10]</a>. For MD simulations, our force field of choice was CHARMM36 <a href="#references">[11]</a> which is well established for protein studies and widely accepted in the scientific community. However, a ligand such as B12 is not a recognized entity in the force field. Thus, it was important to process the topology files separately (for IF and B12) and it served as the bottleneck of our experiment.</p>

        <p class="reveal">One of the most challenging tasks in molecular simulation is the proper processing of ligands. Introducing new entities into the framework of the force field requires careful consideration and validation, often in the form of various quantum mechanical calculations <a href="#references">[12]</a>. This could be a very laboursome and tedious process. For preparing the ligand topology, an automated tool CGenFF <a href="#references">[13]</a> was used. The input file for the tool was generated using another software Avogadro <a href="#references">[14]</a>. An advantage of using CGenFF is that it returns penalty scores for each parameter and our penalty scores were larger than 50, which generally require manual reparametrization. The scores were exceptionally high around the corrin ring and metal ion - cobalt. Thus, we had to resort to other means.</p>   
        <p class="reveal">Due to lack of prior knowledge and time constraints, despite all our efforts, we were unable to complete the model.</p>

      </section>

      <section id="tag2_3">
        <h3 class="revealfromtop">Immunogenicity </h3>
   
        <figure>
          <img src="https://static.igem.wiki/teams/4214/wiki/model/immuno-study-comb.png" class="img-fluid reveal" alt="Immunogenicity predictions with discontinuous epitope">
          <figcaption>
            <p><i style="font-weight:bold;">Figure.</i><i> Immunogenicity predictions with discontinuous epitope with a focus on the region of interest</i></p>
          </figcaption>
        </figure>
        

        <p class="reveal">Since we are modifying the protein, there is a high chance that artificial modifications could trigger an additional immunogenic response. Thus, we decided to screen our candidates for immunogenicity to ensure that the produced mutants do not trigger immunogenic responses themselves. Our focus lies primarily on possible responses from B lymphocytes <a href="#references">[15]</a>.</p>

        <p class="reveal">We performed screens for continuous as well as discontinuous epitopes for possible antibodies of an immunogenetic response. The difference between them is that the discontinuous considers the folding while the linear only takes the ordered amino acids into account <a href="#references">[16]</a>.</p>   
        <p class="reveal">After performing an extensive literature review, we proceeded with the online tool IEDB (Immune Epitope Database and Analysis Resource) <a href="#references">[17]</a>, more specifically the tools ElliPro for discontinuous and Antigen Sequence Properties for the linear epitopes. ElliPro employs a clustering algorithm combined with Thornton's method (focusing on protruding amino acid residues) for making predictions <a href="#references">[18]</a>. The tool Antigen Sequence Properties utilizes several methods (Chou and Fasman beta turn prediction, Emini surface accessibility scale, Karplus and Schulz flexibility scale, Kolaskar and Tongaonkar antigenicity scale, Parker Hydrophilicity Prediction, Bepipred-1.0 Linear Epitope Prediction, and BepiPred-2.0: Sequential B-Cell Epitope Predictor). In short, these methods consider beta turns, surface accessibility probability, mobility of protein segments, physicochemical properties of residues and hydrophilicity. Furthermore, they apply a hidden Markov model, the propensity scale method, and a random forest algorithm.</p>
        <p class="reveal">We expected to get statistically significant differences between wild-type and mutant proteins. However, these expectations were not met for both the linear epitope predictions and the discontinuous ones. For the discontinuous predictions, the wild type caused an immunogenicity score of 202,907 and the mutant with the lowest immunogenicity (a lower score correlates to a lowered predicted chance of autoimmune response) scored 202,578. To conclude, the mutants were not predicted to have significantly altered immunogenicity. This could arise due to several reasons:

          <ol class="reveal">
            <li>
              There is no significant difference in autoantibody binding meaning that the previously described epitope is not the actual one.
            </li>

            <li>The previously described epitope is the actual one. However, our rationale chosen for the mutant design is not applicable to this case. Therefore, mutations with a stronger effect on protein structure should have been selected.
            </li>

            <li>
          Single point mutations introduce very minor changes in structure that do not allow selected tools to detect any significant difference between them. However, it could still be enough to disrupt antibody binding in vitro.
            </li>
          </ol>
        </p>

      </section>

      <section id="references">
        <!-- REFERENCES !!! -->
        <h2 class="revealfromtop">References</h2>
          <div class="tabs reveal">
            <div class="tab">
              <input type="checkbox" id="chck1">
              <label class="tab-label" for="chck1">References</label>
              <div class="tab-content">
                <ol>
                  <li>
                    <p>Guéant JL, Safi A, Aimone-Gastin I, Rabesona H, Bronowicki JP, Plénat F, Bigard MA, and Haertlé T<br>
                    <b style="font-weight: bold !important;">Autoantibodies in pernicious anemia type I patients recognize sequence 251-256 in human intrinsic factor</b><br>
                    <i>Proc Assoc Am Physicians, vol. 109, no. 5, pp. 462-469, 1997</i><br>
                    <a href="https://pubmed.ncbi.nlm.nih.gov/9285945/" target="_blank">"PMID: 9285945"</a>
                    </p>
                    <hr>
                  </li>

                    <li>
                      <p>S Henikoff and J G Henikoff<br>
                      <b style="font-weight: bold !important;">Amino acid substitution matrices from protein blocks</b><br>
                      <i>Proc Natl Acad Sci USA, vol. 89, no. 22, pp. 10915–10919, 1992</i><br>
                      <a href="https://doi.org/10.1073%2Fpnas.89.22.10915" target="_blank"> DOI: 10.1073/pnas.89.22.10915</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>J Jumper, R Evans, A Pritzel, et al.<br>
                      <b style="font-weight: bold !important;">Highly accurate protein structure prediction with AlphaFold</b><br>
                      <i>Nature, vol. 596, pp. 583-589, 2021</i><br>
                      <a href="https://doi.org/10.1038/s41586-021-03819-2" target="_blank">DOI: 10.1038/s41586-021-03819-2</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>M. Mirdita, K. Schütze, Y. Moriwaki, et al.<br>
                      <b style="font-weight: bold !important;">ColabFold: making protein folding accessible to all</b><br>
                      <i>Nat Methods, vol. 19, pp. 679–682, 2022</i><br>
                      <a href="https://doi.org/10.1038/s41592-022-01488-1" target="_blank">DOI: 10.1038/s41592-022-01488-1</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>Schrödinger, LLC<br>
                      <b style="font-weight: bold !important;">The PyMOL Molecular Graphics System, Version 2.5</b><br>
                      <i>Schrödinger, LLC, 2021</i><br>
                      <a href="https://pymol.org/2/#page-top" target="_blank">Read it</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>O. Trott and A.J. Olson<br>
                      <b style="font-weight: bold !important;">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</b><br>
                      <i>J Comput Chem., vol. 31, no. 2, pp. 455-461, 2009</i><br>
                      <a href="https://doi.org/10.1002/jcc.21334" target="_blank">DOI: 10.1002/jcc.21334</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>J. D. Durrant and J. A. McCammon<br>
                      <b style="font-weight: bold !important;">NNScore 2.0: A Neural-Network Receptor–Ligand Scoring Function</b><br>
                      <i>J Chem Inf Model., vol. 51, no. 11, pp. 2897-2903, 2011</i><br>
                      <a href="https://doi.org/10.1021/ci2003889" target="_blank">DOI: 10.1021/ci2003889</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>P. Bauer, B. Hess, and E. Lindahl<br>
                      <b style="font-weight: bold !important;">GROMACS 2022.3 Manual</b><br>
                      <i>Zenodo, 2022</i><br>
                      <a href="https://doi.org/10.5281/zenodo.7037337" target="_blank">DOI: 10.5281/zenodo.7037337</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>N. Guex, and M.C. Peitsch<br>
                      <b style="font-weight: bold !important;">SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modeling</b><br>
                      <i>Electrophoresis, vol. 18, no. 15, pp. 2714-2723, 1997</i><br>
                      <a href="https://doi.org/10.1002/elps.1150181505" target="_blank">DOI: 10.1002/elps.1150181505</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>E.F. Pettersen, T.D. Goddard, C.C Huang, G.S. Couch, D.M. Greenblatt, E.C. Meng, and T.E. Ferrin TE<br>
                      <b style="font-weight: bold !important;">UCSF Chimera--a visualization system for exploratory research and analysis</b><br>
                      <i>J Comput Chem., vol. 25, no. 13, pp. 1605-1612, 2004</i><br>
                      <a href="https://doi.org/10.1002/jcc.20084" target="_blank">DOI: 10.1002/jcc.20084</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>J. Huang, S. Rauscher, G. Nawrocki, T. Ran, M. Feig, B.L. de Groot, H. Grubmuller, and Jr. A.D. MacKerell<br>
                      <b style="font-weight: bold !important;">CHARMM36m: An Improved Force Field for Folded and Intrinsically Disordered Proteins</b><br>
                      <i>Nat Methods, vol. 14, pp. 71-73, 2016</i><br>
                      <a href="http://dx.doi.org/10.1038/nmeth.4067" target="_blank">DOI: 10.1038/nmeth.4067</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>J.A. Lemkul<br>
                      <b style="font-weight: bold !important;">From Proteins to Perturbed Hamiltonians: A Suite of Tutorials for the GROMACS-2018 Molecular Simulation Package, v1.0</b><br>
                      <i>Living J. Comp. Mol. Sci., vol. 1, no. 1, 2018</i><br>
                      <a href="	https://doi.org/10.33011/livecoms.1.1.5068" target="_blank">DOI: 10.33011/livecoms.1.1.5068</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>K. Vanommeslaeghe, E. Hatcher, C. Acharya, S. Kundu, S. Zhong, J. E. Shim, E. Darian, O. Guvench, P. Lopes, I. Vorobyov, and Jr. A.D. MacKerell<br>
                      <b style="font-weight: bold !important;">CHARMM General Force Field (CGenFF): A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</b><br>
                      <i>J. Comput. Chem., vol. 13, no. 4, pp. 671-690, 2010</i><br>
                      <a href="http://dx.doi.org/10.1002/jcc.21367" target="_blank">DOI: 10.1002/jcc.21367</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>M. D. Hanwell, D. E. Curtis, D. C. Lonie, T. Vandermeersch, E. Zurek, and G. R. Hutchison<br>
                      <b style="font-weight: bold !important;">Avogadro: an advanced semantic chemical editor, visualization, and analysis platform</b><br>
                      <i>J. Cheminform., vol. 4, no. 1, p. 17, 2012</i><br>
                      <a href="https://jcheminf.biomedcentral.com/articles/10.1186/1758-2946-4-17" target="_blank">DOI: 10.1186/1758-2946-4-17</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>Johns Hopkins Medicine<br>
                      <b style="font-weight: bold !important;">The Immune System</b><br>
                      <i>Johns Hopkins Medicine</i><br>
                      <a href="https://www.hopkinsmedicine.org/health/conditions-and-diseases/the-immune-system" target="_blank">Read it</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>T. C. Liang<br>
                      <b style="font-weight: bold !important;">Epitopes</b><br>
                      <i>Encyclopedia of Immunology (Second Edition), pp. 825-827, 1998</i><br>
                      <a href="https://doi.org/10.1006/rwei.1999.0219" target="_blank">DOI: 10.1006/rwei.1999.0219</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>R. Vita, S. Mahajan, J. A. Overton, et. al.<br>
                      <b style="font-weight: bold !important;">The Immune Epitope Database (IEDB): 2018 update</b><br>
                      <i>Nucleic Acids Res., vol. 47, no. D1, pp. D339-D343, 2019</i><br>
                      <a href="https://doi.org/10.1093/nar/gky1006" target="_blank">DOI: 10.1093/nar/gky1006</a>
                      </p>
                      <hr>
                    </li>

                    <li>
                      <p>J. Ponomarenko, H.H. Bui, W. Li, N. Fusseder, P.E. Bourne, A. Sette, and B. Peters<br>
                      <b style="font-weight: bold !important;">ElliPro: a new structure-based tool for the prediction of antibody epitopes</b><br>
                      <i>BMC bioinformatics, vol. 9, no. 514, 2008</i><br>
                      <a href="https://pubmed.ncbi.nlm.nih.gov/19055730/" target="_blank">DOI: 10.1186/1471-2105-9-514</a>
                      </p>
                    </li>

                </ol>
              </div>
            </div>  
          </div>
      </section>


      <section>
        <div class="box">
          <a href="{{ url_for('pages', page='dry-lab') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages"><i class="fa fa-solid fa-arrow-left fa_pages"></i> Engineering</p></div></a>

          <a href="{{ url_for('pages', page='software') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages">Software <i class="fa fa-solid fa-arrow-right fa_pages"></i></p></div></a>
        </div>
      </section>
      
    </div>
  </div>
</div>
{% endblock %}
