{% extends "layout.html" %}

<!-- Header -->
{% block header %}
<header class="py-5 mb-5" id="header" style="background-image: linear-gradient(0deg, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%),url(https://static.igem.wiki/teams/4214/wiki/proof-of-concept/proof-of-concept-header.png);">
  <div class="container h-100">
    <div class="row h-100">
      <div class="col-lg-12">
        <h1 class="display-4 text-white">Proof of Concept</h1>
      </div>
    </div>
  </div>
</header>
{% endblock %}

{% block page_content %}

<!-- side bar -->
<div class="row mt-4">
  <div class="col-lg-3">
    <div class="side-bar sticky-top">
      <nav class="navigation">
        <ul class="nav-ul-sidebar">
          <li class="nav-ul-li-sidebar">
            <a class="nav-ul-li-a-sidebar" href="#tag1">Background</a>
            <a class="nav-ul-li-a-sidebar" href="#tag2">Mutation and Plasmid Production in E.coli</a>
            <a class="nav-ul-li-a-sidebar" href="#tag3">Changes in Binding Properties</a>
            <a class="nav-ul-li-a-sidebar" href="#tag4">Immunogenicity</a>
            <a class="nav-ul-li-a-sidebar" href="#tag5">Mammalian cell culture findings</a>
          </li>
        </ul>
      </nav>
    </div>
  </div>
  <!-- END of sidebar -->

  <!-- Include chapters here -->
  <div class="col-lg-9">
    <div class="main-content">
      <section id="tag1">
        <h2 class="revealfromtop">Background</h2>
        <p class="reveal">In order to have a functional Gastric Intrinsic Factor (IF) that could evade the immune system, it first need to carry mutations that would alter its ability to interact with auto-antibodies, while effectively binding to Vitamin B12. For this purpose, the IF gene sequence was first incorporated in a high copy number plasmid that is suitable for high expression and display. This was followed by the introduction of the mutations into gene by site-directed mutagenesis and transformation amplification in bacteria.</p>

        <p class="reveal">The mammalian cell culture, being the final validation tool for our design, was the most important aspect of proof of concept for us! The purpose of that experiment was to investigate how the binding of antibodies differs when comparing our engineered IF (intrinsic factor) to that of the wild type.  As shown in the engineering success page we used an iterative approach with parallel dry lab and recurring wet lab engineering cycles, to confirm the working of our engineered protein. We also took a step towards validating the mutated protein using confirmatory assays and techniques. In addition we also did docking studies in order to check if the mutations would maintain the essential functions for B12 uptake while changing thermodynamic properties for the antibody binding region.</p>
      </section>

      <section id="tag2">
        <h2 class="revealfromtop">Mutation and Plasmid Production in E.coli</h2>
        <p></p>
        <p class="reveal">Through the experiments involving E.coli, we proved that our method and our primers worked for conducting the site-directed mutagenesis and getting the plasmid we desired. In order to achieve this, the plasmids containing the intrinsic factor were first mutated using the primers that we had designed, followed by transformation and sequencing. The sequencing results revealed that the mutations were successful and could be further used for production in mammalian cell lines.<br><br>
The list of functional mutants are as follows:<br><br>
        <div style="padding-left:30px;">
          <i class="fa fa-solid fa-check"></i> IF_001<br>
          <i class="fa fa-solid fa-check"></i> IF_002<br>
          <i class="fa fa-solid fa-check"></i> IF_003<br>
          <i class="fa fa-solid fa-check"></i> IF_101<br>
          <i class="fa fa-solid fa-check"></i> IF_201<br>
          <i class="fa fa-solid fa-check"></i> IF_301<br>
          <i class="fa fa-solid fa-check"></i> IF_203<br>
          <i class="fa fa-solid fa-check"></i> IF_10101<br>
          <i class="fa fa-solid fa-check"></i> IF_20101<br>
          <i class="fa fa-solid fa-check"></i> IF_20201<br>
          <i class="fa fa-solid fa-check"></i> IF_20203
        </div>
        </p>
      </section>

      <section id="tag3">
        <h2 class="revealfromtop">Changes in Binding Properties</h2>
        <p class="reveal">For proof of concept in silico we did docking studies to look at the change in Gibbs free energy. This was done with the help of Autodock Vina and Mutabind2. In order to get the structure predictions of our mutated variants we used Alphafold. We observed changes in Gibbs free energy compared to the wild type. This made us more confident that the mutations didnâ€™t impair structure that is essential for B12 uptake while still differing in antibody binding regions. With autodock vina we concluded that the mutations K258RY262F and S256TK258RY262FV265I showed the best scores. For making sure that the binding properties between cubilin and IF still remained functional we used mutabind2 and concluded that the mutations do not cause any unintended effects with regards to the function. 
        </p>

        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/pdisplay-mammalian-expression-vector2.png" class="img-fluid reveal center" alt="pdisplay-mammalian-expression-vector">
          <figcaption>
            <p><i style="font-weight:bold;">Model Image. </i><i>Mutated IF with B12 docked, mesh structure</i></p>
          </figcaption>
        </figure>

      </section>

      <section id="tag4">
        <h2 class="revealfromtop">Immunogenicity</h2>
        <p class="reveal">We tested immunogenicity in silico, looking at both the continuous and discontinuous epitopes. We decided to screen our candidates for immunogenicity both to make sure that we get the response we want as well as to make sure that the mutagenesis does not raise additional autoimmune responses. The predicted results for our mutants showed almost no difference. We expected to get statistically significant differences between wild-type and mutant proteins. As a concept, the in silico studies would have shown a difference if the structure was significantly changed since it depends on the structural properties.</p>
      </section>

      <section id="tag5">
        <h2 class="revealfromtop">Mammalian cell culture findings</h2>
        <p class="reveal">The expression and surface expression of IF in HEK cells was evaluated by the virtue of epitope specific and unspecific primary antibodies and fluorophore coupled secondary antibodies, through Western Blot, Fluorescence microscopy and Flow cytometry. As mentioned in the experiments sections, the mutant IF sequence was coupled with His tag for protein purification, Myc tag for surface display detection and a PDGFR coupled transmembrane domain for surface display. The Western blot results of the variant 20201 confirmed that the mutated IF protein is expressed properly along with all the tags. The control IF from was detected by our unspecific antibodies. Fluorescence microscopy proved that our method for expression of the mutated IF and detection, works perfectly.  The surface expression was confirmed with the myc specific antibodies. While testing with the epitope specific antibodies, it was observed visually that with increase in the number of mutations, there was a reduction in the intensity of fluorescence signal. This, without image analyses, points in the direction that the IF mutant proteins, thus generated, could infact be able to evade the auto-antibodies. However, the flow cytometry results were inconclusive.</p>
        
        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/proof-of-concept/fluo1.jpg" style="max-width: 700px" class="img-fluid reveal center" alt="pdisplay-mammalian-expression-vector">
          <figcaption>
            <p><i style="font-weight:bold;">Fluorescent Microscopy. </i><i>Image depicting the detection of the gastric IF variant 20201 by the virtue of immunostaining. Red stain shows the presence of PDGFR bound surface IF while the blue colour shows the nuclear staining by DAPI</i></p>
          </figcaption>
        </figure>
      </section>

      <section id="tag6">
        <h2 class="revealfromtop">Future perspectives to address the research question</h2>
        <p class="reveal">
          <div style="padding-left:30px;">
          <i class="fa fa-solid fa-check"></i> Finding the ideal concentration and incubation time for vitamin B12 binding.<br>
          <i class="fa fa-solid fa-check"></i> Utilizing the heightened expression level at 48 hours as supported by the WB.<br>
          <i class="fa fa-solid fa-check"></i> Reconsider and test HEK293 against Chinese Hamster Ovary cell lines for protein production and surface display.<br>
          <i class="fa fa-solid fa-check"></i> Select using Geneticin and refine the selection using sensitive and assay specific secondary antibodies.<br>
          <i class="fa fa-solid fa-check"></i> Sort the displaying cells using a myc coupled magnetic beads technique to concentrate the cells.
          </div>
        </p>
      </section>

      <section>
        <div class="box">
          <a href="{{ url_for('pages', page='results') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages"><i class="fa fa-solid fa-arrow-left fa_pages"></i> Results</p></div></a>

          <a href="{{ url_for('pages', page='implementation') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages">Implementation <i class="fa fa-solid fa-arrow-right fa_pages"></i></p></div></a>
        </div>
      </section>
    </div>
  </div>
</div>
{% endblock %}
