{% extends "layout.html" %}

<!-- Header -->
{% block header %}
<header class="py-5 mb-5" id="header" style="background-image: linear-gradient(0deg, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%),url(https://static.igem.wiki/teams/4214/wiki/results/results-header.jpg);">
  <div class="container h-100">
    <div class="row h-100">
      <div class="col-lg-12">
        <h1 class="display-4 text-white">Results</h1>
      </div>
    </div>
  </div>
</header>
{% endblock %}

{% block page_content %}

<!-- side bar -->
<div class="row mt-4">
  <div class="col-lg-3">
    <div class="side-bar sticky-top">
      <nav class="navigation">
        <ul class="nav-ul-sidebar">
          <li class="nav-ul-li-sidebar">
            <a class="nav-ul-li-a-sidebar" href="#tag0">Results </a>
            <a class="nav-ul-li-a-sidebar" href="#tag1">Bacterial culture results </a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag1_1">pDisplay mammalian expression vector</a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag1_2">PCR amplification of wt-IF gblock fragment</a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag1_3">wt-IF gene into the pDisplay mammalian expression vector </a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag1_4">Site Directed Mutagenesis</a> 
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag1_5">Protein production and purification of wt-IF</a> 

            <a class="nav-ul-li-a-sidebar" href="#tag2">Mammalian cell culture results</a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag2_1">Cell culturing</a>   
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag2_2">Western blot </a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag2_3">Fluorescence microscopy</a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag2_4">Flow cytometry</a>  
            <a class="nav-ul-li-a-sidebar" href="#references">References</a>                  
          </li>
        </ul>
      </nav>
    </div>
  </div>
  <!-- END of sidebar -->

  <!-- Include chapters here -->
  <div class="col-lg-9">
    <div class="main-content">

      <section id="tag0">
        <h2 class="revealfromtop">Results</h2>
       <p class="reveal">
          The results from our experiments are shown on the following page. They are divided into Bacterial culture results and Mammalian cell culture results. The Bacterial part contains our PCR, cloning, sequencing and protein production results. The Mammalian part contains our results from mammalian cell culture western blot, fluorescence microscopy and flow cytometry results. Some abbreviations that you will meet in the text are: wtIF(wild type intrinsic factor), pDisplay mammalian expression vector( the plasmid we bought from ThermoFisher) and Wild-type plasmid is the wtIF gene into the pDisplay mammalian expression vector.</p>
      </section>

      <section id="tag1" style="margin-bottom:30px;">
        <h2 class="revealfromtop">Bacterial culture results</h2>
      </section>

      <section id="tag1_1">
        <h3 class="revealfromtop">pDisplay mammalian expression vector</h3>

        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/pdisplay-mammalian-expression-vector2.png" class="img-fluid reveal center" alt="pdisplay-mammalian-expression-vector">
          <figcaption>
            <p><i style="font-weight:bold;">Agarose gel. </i><i>pDisplay mammalian expression vector. The size of the vector is approximately 5 kb</i></p>
          </figcaption>
        </figure>

        <p class="reveal">We purchased from ThermoFisher Scientific our mammalian expression vector <a href="#references">[1]</a>. pDisplay helps the proteins to be expressed in the surface of mammalian cells. With that way, the interactors of those proteins can be identified with different assays. Some of the features of pDisplay are T7 promoter,SV40 ori, Signal peptide for transmembrane transport, HA tag, Multiple cloning site, Myc tag, PDGFR transmembrane domain, Ampicilin and Neomycin resistance. We amplified the vector in E.coli TOP10 bacteria and performed gel electrophoresis to ensure that we isolate our plasmid. We realised that apart from our vector represented as “Band of interest” detected around 5kb there was Genomic DNA Contamination since another band appeared running higher at 10kb. We proceed with gel extracting only our band of interest in order to clone our gene into the plasmid.</p>
      </section>

      <section id="tag1_2">
        <h3 class="revealfromtop">PCR amplification of wt-IF gblock fragment</h3>

        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/wtif-gblock-in-agarose-gel.png" class="img-fluid reveal center" alt="wtif-gblock-in-agarose-gel">
          <figcaption>
            <p><i style="font-weight:bold;">Agarose gel. </i><i> The wtIF gene from gblock was amplified by using two different set of primers </i></p>
          </figcaption>
        </figure>

        <p class="reveal">We purchased from IDT our wtIF gene with our extra modifications. We included a His-tag for protein production and purification of our protein in bacteria. Therefore, the primers we used to amplify the wtIF gene removed the His-tag and an intrinsic signal peptide that IF had. As it is pictured in the figure above after PCR amplification the size of wtIF gene is around 1.4kb meaning that we amplified our gene.  Our next step then is to perform restriction digestion with BglII and SalI-HF both in the wtIF gene but also in the pDisplay mammalian expression vector.  After this step the two parts were ligated together.</p>
      </section>

      <section id="tag1_3">
        <h3 class="revealfromtop">wt-IF gene into the pDisplay mammalian expression vector</h3>
          
          <figure class="reveal">
            <img src="https://static.igem.wiki/teams/4214/wiki/results/the-wtif-gene-cloned-into-the-pdisplay-mammalian-expression-vector.png" class="img-fluid reveal center" alt="The wtIF gene cloned into the pDisplay mammalian expression vector">
          <figcaption>
            <p><i style="font-weight:bold;">Agarose gel. </i><i> The wtIF gene cloned into the pDisplay mammalian expression vector. The size of the PCR amplified IF fragment is ~1.4kb</i></p>
          </figcaption>
        </figure>

        <p class="reveal">This is the colony PCR agarose gel electrophoresis image of the colonies screened for the wtIF fragment. The plasmids were extracted and amplified using the primers used to amplify the wtIF gene. The lone band in the H10 lane reflects the IF fragment in the solitary clone. This H10 is the clone that wtIF is successfully cloned into pDisplay mammalian expression vector.</p>

        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/wtif-insertion-into-the-pdisplay-mammalian-expression-vector-sequencing-confirmation.png" class="img-fluid reveal center" alt="wtIF insertion into the pDisplay mammalian expression vector sequencing confirmation">
          <figcaption>
            <p><i style="font-weight:bold;">Snapgene picture. </i><i> wtIF insertion into the pDisplay mammalian expression vector sequencing confirmation. The forward and reverse primers used for sequencing aligned on the in-silico created TF_IF_wt plasmid</i></p>
          </figcaption>
        </figure>

        <p class="reveal">We continued with sending the H10 clone for sequencing to ensure that our wtIF gene is inserted succesfully. The sequencing results using the forward and reverse primers yielded good quality ~800bp fragment that aligned with the in silico plasmid. This confirmed the successful cloning of the gene into the plasmid. Therefore since we knew that our gene is there we proceeded with Site Directed Mutagenesis, generating our mutant IFs.</p>
      </section>

      <section id="tag1_4">
        <h3 class="revealfromtop">Site Directed Mutagenesis</h3>

        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/mutant-with-plasmid-backboine.png" class="img-fluid reveal center" alt="mutant with plasmid ">
          <figcaption>
            <p><i style="font-weight:bold;">Snapgene picture. </i><i> The sequencing confirmation of 9 variants (From top left: 201, 20201, 101, 10101, 20101, 203, 20203, 301 and 402)
</i></p>
          </figcaption>
        </figure>
        
        <p class="reveal">These are the mutants that were carried forward for testing our binding assay. We tested the validity of those mutants by sending them for sequencing. In the picture above are shown the primers that we used to create them as well as the sequencing results highlighting the substitutions. The result shows that the necessary substitutions have taken their positions in the mutant variant. Those mutants were amplified in E.coli TOP10 and forwarded for transfection in mammalian cells.</p>
      </section>

      <section id="tag1_5">
        <h3 class="revealfromtop">Protein production and purification of wt-IF</h3>
          <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/wb-bb12-3.png" class="img-fluid reveal center" alt="western blot result">
          <figcaption>
            <p><i style="font-weight:bold;">Western blot. </i><i>Western blot image from the protein production of wtIF in the pSB1C3 plasmid with the lacI promotor after IPTG induction. wtIF is detected around 55-70kDa</i></p>
          </figcaption>
        </figure>
        <p class="reveal">In order to implement our idea and have our protein into a pill, we also attempted to produce and purified the wtIF. The same procedure will be used to produce and purify the best variant from our mutants. For that reason, we used the pSB1C3 Biobrick plasmid with the LacI promotor. This promotor is inducible by IPTG and having the gene of interest, in our case wtIF, under this promotor will drive the protein expression. As a control we used the pDisplay mammalian expression vector and the pSB1C3+LacI without the wtIF. As an extra control we used the Gastric Intrinsic Factor protein so that we ensure that our antibodies detect our protein. After the induction of the protein expression with IPTG we purified our protein. During the purification we collected the cell pellet, the flow through and the eluted part of the purification column. We loaded all of our samples in the SDS-page followed by Western Blot. We used the epitope unspecific Intrinsic Factor(IF) Rabbit primary antibody <a href="#references">[2]</a> and Anti-Rabbit HRP as a secondary antibody. The LacI promotor is induced with 0.5 mM of IPTG  and thus the wtIF is expressed. The wtIF is detected around 55-70kDa in the Lysate and Flow-through of the 0.5 mM IPTG samples but not in the Eluted parts of the purification.</p>
      </section>

      <section id="tag2" style="margin-bottom:30px;">
        <h2 class="revealfromtop">Mammalian cell culture results</h2>
      </section>

      <section id="tag2_1">
        <h3 class="revealfromtop">Cell culturing</h3>

        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/cell-image-3.png" class="img-fluid reveal center" alt="HEK 293T cultured and observed under 100X magnification">
          <figcaption>
            <p><i style="font-weight:bold;">Cell culturing [A and B]. </i><i>HEK 293T cultured and observed under 100X magnification</i></p>
          </figcaption>
        </figure>

        <p class="reveal">The figures A and B successfully establish the wonderful morphology of HEK 293T cells under 100X magnification. The figures A and B are taken at varied locations in different wells. These cultured cells were used for transfection and were further used for validation studies like fluorescent microscopy and western blotting.</p>
      </section>

      <section id="tag2_2">
        <h3 class="revealfromtop">Western blot</h3>

        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/wb.png" class="img-fluid reveal center" alt="western blot result">
          <figcaption>
            <p><i style="font-weight:bold;">Western Blot Image [A and B]. </i><i>Western blot image of our samples-Wild Type, Variant 20201 and Negative control loaded with two different time points (24, 48) and concentrations (15 µg and 25 µg)</i></p>
          </figcaption>
        </figure>

        <p class="reveal">Here, the Anti-Rabbit red fluorescence(1:1000) emitted is from the secondary antibody used against our epitope unspecific Intrinsic Factor(IF) Rabbit primary antibody <a href="#references">[2]</a>. The Anti-Mouse green fluorescence (1:1000) is emitted from the secondary antibody used against our specific Myc Mouse primary antibody <a href="#references">[3]</a>. The red color in the ladder is because of the PageRulerTM Prestained Protein Ladder <a href="#references">[4]</a>. The prestained protein ladder is seen in the 700 nm range under fluorescence and hence they appear red in the blot.</p>
        <p class="reveal">Figure A depicts the Western Blot image (fluorescent image) of our samples-Wild type plasmid, Variant 20201 plasmid and Negative control (without any plasmid) loaded at two different concentrations-15 µg and 25 µg.  The figure also depicts the control housekeeping gene GAPDH at 35 kDa. The samples were also collected at two different time points 24 and 48 hours (transfection time points). The Ladder in kDa is marked along both the sides of the blot. Figure B depicts the same blot in a much enlarged view but without the control GAPDH gene.</p> 
        <p class="reveal">From both Figure A and B, we observed that our protein (marked by green arrows) is being produced in Variant 20201 by the presence of light yellow bands in between 70 kDa to 55 kDa bands. The light yellow bands indicate the binding of both Myc antibody(green) and  epitope unspecific IF primary antibody(red). Myc is present in the plasmid backbone and is one of the tags for confirmation for our protein. However, the streak of green bands seen in Figure B can be understood as unspecified myc binding to various other cellular proteins in HEK cells. There is no red streak of bands seen in wells designated for samples thus indicating that our epitope unspecific antibody is extremely specific to the IF and does not bind to any other cellular proteins.</p>
        <p class="reveal">It is worthy to note that we have no visible bands in negative control as well as in Wild Type plasmid in both figures. One of the reasons for no protein production in WT plasmid is due to removal of the stop codon in the near end of the WT plasmid. While we succeeded in the removal, we believe that it may have caused some changes in the plasmid which we have overlooked by not sequencing the entire plasmid.</p>
      </section>

      <section id="tag2_3">
        <h3 class="revealfromtop">Fluorescence microscopy</h3>
        <figure class="reveal">
          <img src="https://static.igem.wiki/teams/4214/wiki/results/fm.png" class="img-fluid reveal center" alt="fluorescence microscopy">
          <figcaption>
            <p><i style="font-weight:bold;">Fluorescence microscopy. </i><i>Figures A to F represent the Fluorescence microscope images of HEK 293T cells. Scale is marked at the bottom left of the image. A: Negative control in Bright field B: Negative control overlapped with Bright field and DAPI staining, C: Positive control in GFP channel, D: Positive control GFP superimposed with  DAPI staining, E: Variant 20201 in RFP channel, F: Variant 20201 superimposed with IF and DAPI staining.
The images were analyzed using FIJI-J imaging software <a href="#references">[5]</a>.
</i></p>
          </figcaption>
        </figure>
        <p class="reveal">From Figure A we can confirm that our cells are adherent and in good morphology. From Figure B, with the superimposition, we can establish that the blue DAPI (4′,6-diamidino-2-phenylindole) staining worked and every cell appeared to be stained . In Figure C, we can see the Green Fluorescence Protein (GFP-which is the positive control for the transfection), confirming that our transfection worked. In Figure D, superimposing the GFP with blue DAPI was done to see the transfection efficiency visually. As one can observe, the transfection efficiency is not very high for these cells. In Figure E, in the red fluorescence spectrum, we can see our IF protein. These IF proteins were tagged with primary rabbit Anti IF epitope unspecific antibody (1:100) and with secondary anti rabbit in red fluorescence (1:400). This confirms that our IF protein is expressed, thus coinciding with our western blot results as well. Figure F displays the superimposed image of DAPI staining the nuclei (blue) and the expression of IF on the surface of Variant 20201 (red).</p>
      </section>

      <section id="tag2_4">
        <h3 class="revealfromtop">Flow cytometry</h3>
        <p class="reveal">We also performed the flow cytometry assay in HEK293 suspension cells. The results though weren’t conclusive. The positive control was expressed and gave some signal but neither the Wild-type plasmid nor the mutants were expressed.</p>
      </section>


      <section id="references">
        <!-- REFERENCES !!! -->
        <h2 class="revealfromtop">References</h2>
          <div class="tabs reveal">
            <div class="tab">
              <input type="checkbox" id="chck1">
              <label class="tab-label" for="chck1">References</label>
              <div class="tab-content">
                <ol>
                  <li>
                    <p>ThermoFisher Scientific<br>
                    <b style="font-weight: bold !important;">pDisplay™ Mammalian Expression Vector</b><br>
                    <i>ThermoFisher Scientific</i><br>
                    <a href="https://www.thermofisher.com/order/catalog/product/V66020" target="_blank">Read it</a>
                    </p>
                    <hr>
                  </li>

                  <li>
                    <p>Atlas Antibodies<br>
                    <b style="font-weight: bold !important;">Polyclonal Anti-GIF Antibody</b><br>
                    <i>Atlas Antibodies</i><br>
                    <a href="https://www.atlasantibodies.com/products/antibodies/primary-antibodies/triple-a-polyclonals/gif-antibody-hpa039908/?q=&t= " target="_blank">Read it</a>
                    </p>
                    <hr>
                  </li>

                  <li>
                    <p>ThermoFisher Scientific<br>
                    <b style="font-weight: bold !important;">c-Myc Monoclonal Antibody (9E10)</b><br>
                    <i>ThermoFisher Scientific</i><br>
                    <a href="https://www.thermofisher.com/antibody/product/c-Myc-Antibody-clone-9E10-Monoclonal/MA1-980?imageId=28579" target="_blank">Read it</a>
                    </p>
                    <hr>
                  </li>

                  <li>
                    <p>ThermoFisher Scientific<br>
                    <b style="font-weight: bold !important;">PageRuler™ Prestained Protein Ladder, 10 to 180 kDa</b><br>
                    <i>ThermoFisher Scientific</i><br>
                    <a href="https://www.thermofisher.com/order/catalog/product/26616" target="_blank">Read it</a>
                    </p>
                    <hr>
                  </li>

                  <li>
                    <p>ImageJ Schneider, C. A., Rasband, W. S., & Eliceiri, K. W. (2012)<br>
                    <b style="font-weight: bold !important;">NIH Image to ImageJ: 25 years of image analysis.</b><br>
                    <i>Nature Methods, 9(7), 671–675</i><br>
                    <a href="https://imagej.net/software/imagej/" target="_blank">Read it</a>
                    </p>
                  </li>
                </ol>
              </div>
            </div>  
          </div>
      </section>


      <section>
        <div class="box">
          <a href="{{ url_for('pages', page='parts') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages"><i class="fa fa-solid fa-arrow-left fa_pages"></i> Parts</p></div></a>

          <a href="{{ url_for('pages', page='proof-of-concept') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages">Proof of concept <i class="fa fa-solid fa-arrow-right fa_pages"></i></p></div></a>
        </div>
      </section>
    </div>
  </div>
</div>
{% endblock %}
