{% extends "layout.html" %}

<!-- Header -->
{% block header %}
<header class="py-5 mb-5" id="header" style="background-image: linear-gradient(0deg, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%),url(https://static.igem.wiki/teams/4214/wiki/software/software-header.png);">
  <div class="container h-100">
    <div class="row h-100">
      <div class="col-lg-12">
        <h1 class="display-4 text-white">Software</h1>
      </div>
    </div>
  </div>
</header>
{% endblock %}

{% block page_content %}

<!-- side bar -->
<div class="row mt-4">
  <div class="col-lg-3">
    <div class="side-bar sticky-top">
      <nav class="navigation">
        <ul class="nav-ul-sidebar">
          <li class="nav-ul-li-sidebar">
            <a class="nav-ul-li-a-sidebar" href="#tag1">Background</a>
            <a class="nav-ul-li-a-sidebar" href="#tag2">Description</a>
            <a class="nav-ul-li-a-sidebar" href="#tag3">Tutorials</a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag3_1">Screening mode</a>
                <a class="nav-ul-li-a-sidebar light_sidebar" href="#tag3_2">Mutants generation</a>
            <a class="nav-ul-li-a-sidebar" href="#references">References</a>
          </li>
        </ul>
      </nav>
    </div>
  </div>
  <!-- END of sidebar -->

  <!-- Include chapters here -->
  <div class="col-lg-9">
    <div class="main-content">

      <section id="tag1">
        <h2>Background</h2>
          <p>Protein design is essential for any protein engineering project. While designing new proteins is still challenging, improvement of existing ones by using site-directed mutagenesis seems to be feasible. As the amount of mutation sites space is huge, an in silico preselection of mutants is needed. Several tools are available that enable the prediction of the mutation effect on protein structures. However, they are complicated to use and they do not give any suggestions with which mutants to proceed in a particular experiment <a href="#references">[1]</a><a href="#references">[2]</a>.</p>

          <p class="reveal">In order to address this problem, we generated the powerful software ProMutor.</p>

          <p class="reveal">ProMutor is a flexible, easy-to-use, open-source platform for generating advantageous point mutants. In our software, wild-type sequences are used to suggest a list of potent mutations based on explicit modelling of evolutionary history. It enables the user to pick the variants of interest based on epistatic scores. To our knowledge, it is the first web-based software to tackle the complexity of protein design.</p>
      </section>

      <section id="tag2">
        <h2 class="revealfromtop">Description</h2>

        <p class="reveal">We developed a web-based user-friendly tool to facilitate protein design. As a starting point, our tool takes either nucleotide or amino acid sequences. Multiple operation modes are offered to the client to make the process of mutant creation as customised as possible. Based on the desired protein design settings, ProMutor provides a list of mutants with the predicted effect of introduced mutations based on the epistatic effect and amino acid conservation across various species.</p>

        <p class="reveal">The mutant generation pipeline begins with a wild-type sequence and a list of positions to be mutated. For instance, this could be a known epitope that is to be modified, post-translational modification sites that should be removed, etc. In addition, our tool could be used as a screening instrument for the effect of all possible single-point mutations. This would facilitate the screening for amino acids that are essential for the protein’s structure and function.</p>

        <figure>
          <img src="https://static.igem.wiki/teams/4214/wiki/software/promutor-scheme.png" class="img-fluid center reveal" alt="heatmap">
          <figcaption>
            <p><i style="font-weight:bold;">Figure.</i><i> Schematic representation of the ProMutor pipeline</i></p>
          </figcaption>
        </figure>

        <p class="reveal">Two modes enable various types of mutant generation. Once a mutation list is submitted, variants with either a similar WT structure (“Preserved” mode of the tool) or a significantly differing one (“Disrupted” mode of the tool) are designed. In both cases, substitution matrices such as BLOSUM or PAM <a href="#references">[3]</a> are utilised for a pre-selection of amino acids. A wide choice of those (8) is provided to the user so that the prediction could be as tunable as possible. In cases of conserving structures, most similar amino acids are chosen as substitutes since it's supposed that evolutionary-tolerated amino acid substitutions cause neither major perturbations on the protein's structure nor on its function. On the contrary, when the protein structure should be completely disrupted, the mutations that are least tolerated in evolution are selected to proceed with.</p>

        <p class="reveal">After compiling a list of mutants for testing, ProMutor evaluates the effect of the proposed interventions by predicting their mutational outcomes. For this purpose, a software called GEMME <a href="#references">[4]</a><a href="#references">[5]</a> is used, which evaluates mutations based on explicit modelling of the evolutionary history of natural sequences. In addition, GEMME is much faster than other existing tools. It takes into account only biologically meaningful and interpretable parameters and does not require any complex learning steps. Given input alignment, it generates a complete landscape of protein mutations in minutes. Extensive validation was performed to show that GEMME is on par or better than more complex tools. Moreover, GEMME could handle combinations of mutations and give reliable predictions for those.</p>

        <p class="reveal">For our purposes, HHblits <a href="#references">[6]</a> and Uniclust <a href="#references">[7]</a> database are chosen to create a multiple sequence alignment (MSA) required as an input for GEMME. By providing several databases (Uniclust, PDB70 <a href="#references">[8]</a>, PfamA <a href="#references">[9]</a>, SCOP70 <a href="#references">[10]</a>), the user has the freedom to choose. However, we suggest Uniclust database due to greater search space and more reliable predictions.</p>

        <p class="reveal">Once all steps are complete, the user gets a list of scored point mutants as well as their sequences. In order to enable reproducibility, all inputs and outputs are interpreted and reported. The results can be downloaded and displayed in several formats for easy usage. Last but not least, the user is provided with protein prediction possibilities as a follow-up step (ColabFold <a href="#references">[11]</a><a href="#references">[12]</a>).</p>

        <p class="reveal">Overall, this sophisticated pipeline is a multiple steps process which is fully automated without any user involvement. Calculations are performed on a powerful multi-core cluster and accessible through a single web form.</p>

        <p class="reveal">Note from authors: “Our vision is to provide a complete and powerful tool that makes protein design as painless as possible.”</p>
      </section>

      <section id="tag3">
        <h2 class="revealfromtop">Tutorials</h2>
        <p class="reveal">This tutorial explains how to set every parameters in order to efficiently use both the mode that are available (screening and mutants generation). For more detailed explanations, please visit the <a href="https://gitlab.igem.org/2022/software-tools/stockholm" target="_blank">Gitlab</a>.</p>
      </section>

      <section id="tag3_1" class="reveal">
        <h3>Screening mode</h3>
        <h4>Input</h4>
        <h5>Step 1: Choose your sequence type</h5>
        <p>Is it a DNA (nucleotide bases) or Protein (amino acids) sequence?</p>
        <p style="margin-bottom:60px;">
        <code>
protein
        </code>
        </p>

        <h5>Step 2: Insert your Sequence</h5>
        <p>You can choose to upload a fasta file (.fasta, .fa, .fna, .fnn, .faa) or paste the sequence as follows:</p>

        <div style="background-color: whitesmoke; border-radius: .2rem; font-style:monospace; text-align: justify; text-align-last: justify; opacity:0.75;">
      <p style="margin-bottom:60px;"><code>   
>sp|P27352|IF_HUMAN Cobalamin binding intrinsic factor OS=Homo sapiens OX=9606 GN=CBLIF PE=1 SV=2<br>
MAWFALYLLSLLWATAGTSTQTQSSCSVPSAQEPLVNGIQVLMENSVTSSAYPNPSILIA
MNLAGAYNLKAQKLLTYQLMSSDNNDLTIGQLGLTIMALTSSCRDPGDKVSILQRQMENW
APSSPNAEASAFYGPSLAILALCQKNSEATLPIAVRFAKTLLANSSPFNVDTGAMATLAL
TCMYNKIPVGSEEGYRSLFGQVLKDIVEKISMKIKDNGIIGDIYSTGLAMQALSVTPEPS
KKEWNCKKTTDMILNEIKQGKFHNPMSIAQILPSLKGKTYLDVPQVTCSPDHEVQPTLPS
NPGPGPTSASNITVIYTINNQLRGVELLFNETINVSVKSGSVLLVVLEEAQRKNPMFKFE
TTMTSWGLVVSSINNIAENVNHKTYWQFLSGVTPLNEGVADYIPFNHEHITANFTQY
      </code></p>
        </div>


        <h5>Step 3: Insert the position of mutation(s)</h5>
        <p>Set the position to 0 to enable screening mode</p>
        <p style="margin-bottom:60px;"><code>
0
        </code></p>

        <h5>Step 4: Insert the positions that are always of mutated</h5>
        <p style="margin-bottom:60px;"><code>
Leave empty
        </code></p>
 

        <h5>Step 5: Insert maximum number of mutation(s)</h5>
        <p style="margin-bottom:60px;"> <code>
Leave empty
        </code></p>
 

        <h5>Step 6: Choose structure preservation mode</h5>
        <p style="margin-bottom:60px;">By default: <code>
Preserved
        </code></p>


        <h5>Step 7: Choose the similarity matrix</h5>
        <p style="margin-bottom:60px;">By default: <code>
Blosum62
        </code></p>


        <h5>Step 8: Choose the database to be used</h5>
        <p>Select which database should be used to construct an MSA. Uniclust is recommended for more reliable predictions. Faster results may be produced with other choices.</p>
        <p style="margin-bottom:60px;"><code>
Uniclust
        </code></p>


        <h5>Step 9: Submit the form</h5>
        <p style="margin-bottom:60px;">Press: <code>
Run
        </code></p>


        <p>Or you can watch the video below:</p>
        <iframe title="Stockholm: ProMutor Screening Mode Tutorial (2022) [English]" width="560" height="315" src="https://video.igem.org/videos/embed/5c18bfd4-3851-4d7f-aac4-997cef7eee01" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups" class="center"></iframe>


        <h4>Output</h4>
        <h5>Selected parameters</h5>

        <table>
          <thead>
            <tr>
              <th>Argument</th>
              <th>Chosen</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>Sequence</td>
              <td>&gt;sp|P27352|IF_HUMAN Cobalamin binding intrinsic factor OS=Homo sapiens OX=9606 G N=CBLIF PE=1 SV=2<br> MAWFALYLLSLLWATAGTSTQTQSSCSVPSAQEPLVNGIQVLMENSVTSSAYPNPSILIA MNLAGAYNLKAQKLLTYQLMSSDNNDLTIGQLGLTIMALTSSCRDPGDKVSILQRQMENW APSSPNAEASAFYGPSLAILALCQKNSEATLPIAVRFAKTLLANSSPFNVDTGAMATLAL TCMYNKIPVGSEEGYRSLFGQVLKDIVEKISMKIKDNGIIGDIYSTGLAMQALSVTPEPS KKEWNCKKTTDMILNEIKQGKFHNPMSIAQILPSLKGKTYLDVPQVTCSPDHEVQPTLPS NPGPGPTSASNITVIYTINNQLRGVELLFNETINVSVKSGSVLLVVLEEAQRKNPMFKFE TTMTSWGLVVSSINNIAENVNHKTYWQFLSGVTPLNEGVADYIPFNHEHITANFTQY</td>
            </tr>
            <tr>
              <td>Matrix</td>
              <td>Blosum62</td>
            </tr>
            <tr>
              <td>Database</td>
              <td>uniclust</td>
            </tr>
            <tr>
              <td>Number of mutant</td>
              <td></td>
            </tr>
            <tr>
              <td>Positions</td>
              <td>0</td>
            </tr>
            <tr>
              <td>Preserved</td>
              <td>True</td>
            </tr>
            <tr>
              <td>Input type</td>
              <td>protein</td>
            </tr>
          </tbody>
        </table>


        <h5 style="margin-top:60px;">Screening result</h5>
        <p>The results of the screening are displayed on a Heatmap Figure. Amino acids positions of the protein are represented on the x-axis while each possible conservation score is presented on the y-axis. 
          The resulting <i><span>&#916;</span><span>&#916;</span>E</i> scores indicate evolutionary tolerance of all possible substitutions per position (y-axis). Here, highly variable positions have <i><span>&#916;</span><span>&#916;</span>E</i> ≈ 0 (have minor effect on protein structure) and more deleterious substitutions get lower <i><span>&#916;</span><span>&#916;</span>E</i> scores. Simply, this plot depicts the mutations effect on protein stucture with red and blue corresponding to low and high effect, respectively.
        </p>

        <figure>
          <img src="https://static.igem.wiki/teams/4214/wiki/software/if-software-plot1.png" class="img-fluid center" alt="heatmap">
          <figcaption>
            <p><i style="font-weight:bold;">Figure.</i><i> Example of the Heatmap results one should expect using the screening mode with the amino acids position on the x-axis and the evolutionary scores for every amino acids at each respective positions on the y-axis</i></p>
          </figcaption>
        </figure>

      </section>

      <section id="tag3_2" class="reveal">
        <h3>Mutants generation</h3>
        <h4>Input</h4>
        <h5>Step 1: Choose your sequence type</h5>
        <p>Is it a DNA (nucleotide bases) or Protein (amino acids) sequence?</p>
        <p style="margin-bottom:60px;">
        <code>
protein
        </code>
        </p>

        <h5>Step 2: Insert your Sequence</h5>
        <p>You can choose to upload a fasta file (.fasta, .fa, .fna, .fnn, .faa) or paste the sequence as follows:</p>

        <div style="background-color: whitesmoke; border-radius: .2rem; font-style:monospace; text-align: justify; text-align-last: justify; opacity:0.75;">
        <p style="margin-bottom:60px;"><code>   
>sp|P27352|IF_HUMAN Cobalamin binding intrinsic factor OS=Homo sapiens OX=9606 GN=CBLIF PE=1 SV=2<br>
MAWFALYLLSLLWATAGTSTQTQSSCSVPSAQEPLVNGIQVLMENSVTSSAYPNPSILIA
MNLAGAYNLKAQKLLTYQLMSSDNNDLTIGQLGLTIMALTSSCRDPGDKVSILQRQMENW
APSSPNAEASAFYGPSLAILALCQKNSEATLPIAVRFAKTLLANSSPFNVDTGAMATLAL
TCMYNKIPVGSEEGYRSLFGQVLKDIVEKISMKIKDNGIIGDIYSTGLAMQALSVTPEPS
KKEWNCKKTTDMILNEIKQGKFHNPMSIAQILPSLKGKTYLDVPQVTCSPDHEVQPTLPS
NPGPGPTSASNITVIYTINNQLRGVELLFNETINVSVKSGSVLLVVLEEAQRKNPMFKFE
TTMTSWGLVVSSINNIAENVNHKTYWQFLSGVTPLNEGVADYIPFNHEHITANFTQY
        </code></p>
        </div>

        <h5>Step 3: Insert the position of mutation(s)</h5>
        <p>You can decide at which position you want to insert mutations.</p>
        <p style="margin-bottom:60px;"><code>
28, 57, 97
        </code></p>

        <h5>Step 4: Insert the positions that are always of mutated</h5>
        <p>If a position from the <i>STEP 3</i> has to be inserted in each generated sequence, you can specify it. Leave empty for a combinatory mix.</p>
        <p style="margin-bottom:60px;"><code>
57
        </code></p>
 

        <h5>Step 5: Insert maximum number of mutation(s)</h5>
        <p>Based on the postions from <i>STEP 3</i> you can choose the maximum number of mutations pre sequence to include. In this tutorial we choose postions 28, 57, 97 (so 3 different positions), therefore, we can decide to include up to 3 mutations per sequence. With the maximum number of mutations set to 3, the software generates singlets, doublets and triplets.</p>
        <p style="margin-bottom:60px;"><code>
3
        </code></p>
 

        <h5>Step 6: Choose structure preservation mode</h5>
        <p>You can choose whether you want mutants with preserved or disrupted structure depending on the purpose. Preserved mode favoritise similar amino acids while disrupted favoritise more distant ones.</p>
        <p style="margin-bottom:60px;"><code>
Preserved
        </code></p>


        <h5>Step 7: Choose the similarity matrix</h5>
        <p>Depending on the divergence or pairwise identity desired, you can select among Blosum45, Blosum50, Blosum62, Blosum80, Blosum90 or Pam30, Pam90 or even Pam250. By default, Blosum62 is selected.</p>
        <p style="margin-bottom:60px;"><code>
Blosum62
        </code></p>


        <h5>Step 8: Choose the database to be used</h5>
        <p>Select against which database the MSA should be ran. Uniclust is recommended for more reliable prediction. Faster results may be produced with other choices.</p>
        <p style="margin-bottom:60px;"><code>
Uniclust
        </code></p>


        <h5>Step 9: Submit the form</h5>
        <p style="margin-bottom:60px;">Press: <code>
Run
        </code></p>

      <p>Or you can watch the video below:</p>
      <iframe title="Stockholm: ProMutor Mutant Generation Tutorial (2022) [English]" width="560" height="315" src="https://video.igem.org/videos/embed/baf7494a-87da-4603-bbb2-a70972c1c3da" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups" class="center"></iframe>


        <h4>Output</h4>
        <h5>Selected parameters</h5>

        <table>
          <thead>
            <tr>
              <th>Argument</th>
              <th>Chosen</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>Sequence</td>
              <td>&gt;sp|P27352|IF_HUMAN Cobalamin binding intrinsic factor OS=Homo sapiens OX=9606 G N=CBLIF PE=1 SV=2<br> MAWFALYLLSLLWATAGTSTQTQSSCSVPSAQEPLVNGIQVLMENSVTSSAYPNPSILIA MNLAGAYNLKAQKLLTYQLMSSDNNDLTIGQLGLTIMALTSSCRDPGDKVSILQRQMENW APSSPNAEASAFYGPSLAILALCQKNSEATLPIAVRFAKTLLANSSPFNVDTGAMATLAL TCMYNKIPVGSEEGYRSLFGQVLKDIVEKISMKIKDNGIIGDIYSTGLAMQALSVTPEPS KKEWNCKKTTDMILNEIKQGKFHNPMSIAQILPSLKGKTYLDVPQVTCSPDHEVQPTLPS NPGPGPTSASNITVIYTINNQLRGVELLFNETINVSVKSGSVLLVVLEEAQRKNPMFKFE TTMTSWGLVVSSINNIAENVNHKTYWQFLSGVTPLNEGVADYIPFNHEHITANFTQY</td>
            </tr>
            <tr>
              <td>Matrix</td>
              <td>Blosum62</td>
            </tr>
            <tr>
              <td>Database</td>
              <td>uniclust</td>
            </tr>
            <tr>
              <td>Number of mutant</td>
              <td>3</td>
            </tr>
            <tr>
              <td>Positions</td>
              <td>28,57,97</td>
            </tr>
            <tr>
              <td>Preserved</td>
              <td>True</td>
            </tr>
            <tr>
              <td>Input type</td>
              <td>protein</td>
            </tr>
          </tbody>
        </table>

        <h5 style="margin-top:60px;">Mutant sequences and scores</h5>
        <p>The mutant names are given together with their respective score in a table like below.<br>
        <i>Sample output:</i></p>

        <table>
          <thead>
            <tr>
              <th>Scores:</th>
              <th>Mutant Score</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>I57V,M97L</td>
              <td>-0.623305422209993</td>
            </tr>
            <tr>
              <td>V28I,I57V</td>
              <td>-0.867803185985955</td>
            </tr>
            <tr>
              <td>V28I,I57V,M97L</td>
              <td>-1.2911568830377</td>
            </tr>
          </tbody>
        </table>

        <p style="margin-top:50px;">Different functions are then available to display or download freshly generated sequences. Use the dropdown toggle to select a mutant. Then, press the button <code>Show sequence</code> to display its sequence. You can also open the sequence in a new tab using the button <code>Open in new tab</code>. Every generated sequence is separately downloadable in fasta format by selecting it from the dropdown and pressing <code>Download fasta</code>. If every mutant sequences have to be downloaded in a single fasta file, use the button <code>Download every fasta</code>.</p>

      </section>

        <div class="box_page">
          <a href="https://promutor.com/" target="_blank" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages">ProMutor <svg fill="#FFFFFF" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc. --><path d="M438.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-160-160c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L338.8 224 32 224c-17.7 0-32 14.3-32 32s14.3 32 32 32l306.7 0L233.4 393.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0l160-160z"/></svg></p></div></a>
        </div>
      </section>


      <section id="references">
        <!-- REFERENCES !!! -->
        <h2 class="revealfromtop">References</h2>
          <div class="tabs reveal">
            <div class="tab">
              <input type="checkbox" id="chck1">
              <label class="tab-label" for="chck1">References</label>
              <div class="tab-content">
                <ol>
                  <li>
                    <p><!--authors here-->J.-E. Shin, A. J. Riesselman, A. W. Kollasch, C. McMahon, E. Simon, C. Sander, A. Manglik, A. C. Kruse, and D. S. Marks<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Protein design and variant prediction using autoregressive generative models</b><br>
                    <!-- journal's name--><i>Nature communications, vol. 12, no. 1, pp. 1–11, 2021.</i><br>
                    <!-- link to artile--><a href="https://www.biorxiv.org/content/10.1101/757252v2" target="_blank">DOI: 10.1101/757252</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->A. J. Riesselman, J. B. Ingraham, and D. S. Marks<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Deep generative models of genetic
variation capture mutation effects</b><br>
                    <!-- journal's name--><i>bioRxiv, 2017</i><br>
                    <!-- link to artile--><a href="https://www.biorxiv.org/content/10.1101/235655v2" target="_blank">DOI: 10.1101/235655</a>
                    </p>
                  <hr>
                  </li>
                  
                  <li>
                    <p><!--authors here-->S. Henikoff and J. G. Henikoff<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Amino acid substitution matrics from protein blocks.</b><br>
                    <!-- journal's name--><i>Proc Natl Acad Sci USA, vol. 89, 1992.</i><br>
                    <!-- link to artile--><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC50453/" target="_blank">DOI: 10.1073/pnas.89.22.10915</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->E. Laine, Y. Karami, and A. Carbone<br>
                    <!--paper titles--><b style="font-weight: bold !important;">GEMME: A Simple and Fast Global Epistatic
Model Predicting Mutational Effects</b><br>
                    <!-- journal's name--><i>Molecular Biology and Evolution, vol. 36, pp. 2604–
2619, 08 2019.</i><br>
                    <!-- link to artile--><a href="https://academic.oup.com/mbe/article/36/11/2604/5548199" target="_blank">DOI: 10.1093/molbev/msz179</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->M. H. Høie, M. Cagiada, A. H. Beck Frederiksen, A. Stein, and K. Lindorff-Larsen<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Predicting
and interpreting large-scale mutagenesis data using analyses of protein stability
and conservation.</b><br>
                    <!-- journal's name--><i>Cell Reports, vol. 38, no. 2, p. 110207, 2022.</i><br>
                    <!-- link to artile--><a href="https://www.sciencedirect.com/science/article/pii/S2211124721017113" target="_blank">DOI: 10.1016/j.celrep.2021.110207</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->M. Remmert, A. Biegert, A. Hauser, and J. Söding<br>
                    <!--paper titles--><b style="font-weight: bold !important;">HHblits: lightning-fast iterative protein
sequence searching by HMM-HMM alignment</b><br>
                    <!-- journal's name--><i>Nature Methods, vol. 9, pp. 173–175,
Feb. 2012.</i><br>
                    <!-- link to artile--><a href="https://www.nature.com/articles/nmeth.1818" target="_blank">DOI: 10.1038/nmeth.1818</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->M. Mirdita, L. von den Driesch, C. Galiez, M. J. Martin, J. Söding, and M. Steinegger<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Uniclust databases of clustered and deeply annotated protein sequences and alignments</b><br>
                    <!-- journal's name--><i>Nucleic Acids Res, vol. 45, pp. D170–D176, Nov. 2016.</i><br>
                    <!-- link to artile--><a href="https://pubmed.ncbi.nlm.nih.gov/27899574/" target="_blank">DOI: 10.1093/nar/gkw1081</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->M. Steinegger, M. Meier, M. Mirdita, H. Vöhringer, S. J. Haunsberger, and J. Söding<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Hh-suite3 for fast remote homology detection and deep protein annotation</b><br>
                    <!-- journal's name--><i>BMC Bioinformatics, vol. 20, p. 473, Sep 2019.</i><br>
                    <!-- link to artile--><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3019-7" target="_blank">DOI: 10.1186/s12859-019-3019-7</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->E. L. Sonnhammer, S. R. Eddy, and R. Durbin<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Pfam: a comprehensive database of protein
domain families based on seed alignments</b><br>
                    <!-- journal's name--><i>Proteins, vol. 28, pp. 405–420, July 1997.</i><br>
                    <!-- link to artile--><a href="https://pubmed.ncbi.nlm.nih.gov/9223186/" target="_blank">DOI: 10.1002/(sici)1097-0134</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->A. G. Murzin, S. E. Brenner, T. Hubbard, and C. Chothia<br>
                    <!--paper titles--><b style="font-weight: bold !important;">SCOP: a structural classification
of proteins database for the investigation of sequences and structures</b><br>
                    <!-- journal's name--><i>J Mol Biol, vol. 247, pp. 536–540, Apr. 1995.</i><br>
                    <!-- link to artile--><a href="https://pubmed.ncbi.nlm.nih.gov/7723011/" target="_blank">DOI: 10.1006/jmbi.1995.0159</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->J. Jumper, R. Evans, A. Pritzel, T. Green, M. Figurnov, O. Ronneberger, K. Tunyasuvunakool, R. Bates, A. Žídek, A. Potapenko, A. Bridgland, C. Meyer, S. A. A. Kohl, A. J.
Ballard, A. Cowie, B. Romera-Paredes, S. Nikolov, R. Jain, J. Adler, T. Back, S. Petersen,
D. Reiman, E. Clancy, M. Zielinski, M. Steinegger, M. Pacholska, T. Berghammer, S. Bodenstein,
D. Silver, O. Vinyals, A. W. Senior, K. Kavukcuoglu, P. Kohli, and D. Hassabis<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Highly accurate protein structure prediction with alphafold</b><br>
                    <!-- journal's name--><i>Nature, vol. 596, pp. 583–589, Aug 2021.</i><br>
                    <!-- link to artile--><a href="https://www.nature.com/articles/s41586-021-03819-2" target="_blank">DOI: 10.1038/s41586-021-03819-2</a>
                    </p>
                  <hr>
                  </li>

                  <li>
                    <p><!--authors here-->M. Mirdita, K. Schütze, Y. Moriwaki, L. Heo, S. Ovchinnikov, and M. Steinegger<br>
                    <!--paper titles--><b style="font-weight: bold !important;">Colabfold:
making protein folding accessible to all.</b><br>
                    <!-- journal's name--><i>Nature Methods, vol. 19, pp. 679–682, Jun 2022.</i><br>
                    <!-- link to artile--><a href="https://www.nature.com/articles/s41592-022-01488-1" target="_blank">DOI: 10.1038/s41592-022-01488-1</a>
                    </p>
                  </li>
                </ol>
              </div>
            </div>  
          </div>
      </section>


      <section>
        <div class="box">
          <a href="{{ url_for('pages', page='model') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages"><svg fill="#FFFFFF" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc. --><path d="M9.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.2 288 416 288c17.7 0 32-14.3 32-32s-14.3-32-32-32l-306.7 0L214.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z"/></svg> Model</p></div></a>

          <a href="{{ url_for('pages', page='experiments') }}" class="a_pages"><div class="reveal flashbox_pages"><p class="p_pages">Experiments <svg fill="#FFFFFF" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc. --><path d="M438.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-160-160c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L338.8 224 32 224c-17.7 0-32 14.3-32 32s14.3 32 32 32l306.7 0L233.4 393.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0l160-160z"/></svg></p></div></a>
        </div>
      </section>
    </div>
  </div>
</div>
{% endblock %}
